cmd/workspace/grants/grants.go

```diff
@@ -19,7 +19,7 @@ var cmdOverrides []func(*cobra.Command)
 func New() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "grants",
-		Short: `In Unity Catalog, data is secure by default.`,
+		Short: `Manage privileges granted in Unity Catalog.`,
```

Comment:
It's an auto-generated code so this should be changed in OpenAPI

Reply:
Is it? That wasn't obvious to me. Shouldn't these be in a separate directory that e.g. says __generated?

Reply:
All the CLI commands are generated and have "auto generated" comment at the top of the file which is usual Go lang practice